launcher/Application.cpp

```diff
@@ -394,7 +394,11 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
 
     // init the logger
     {
-        static const QString logBase = BuildConfig.LAUNCHER_NAME + "-%0.log";
+        static const QString logBase = "logs/"+BuildConfig.LAUNCHER_NAME + "-%0.log";
+        QDir logDir = QDir(dataPath);
+        if(!logDir.exists("logs")) {
+            logDir.mkpath("logs"); //this can fail, but there is no need to throw an error *yet*, since it also triggers the error message below!
+        }
```

Comment:
```suggestion
        static const QString logBase = FS::PathCombine("logs", BuildConfig.LAUNCHER_NAME + "-%0.log");
        FS::ensureFolderPathExists("logs"); // this can fail, but there is no need to throw an error *yet*, since it also triggers the error message below!
```
The launcher changes its working directory to the data path so we can just use a relative path here instead